agents/matlab/matlab_agent/test/unit/test_connect.py

```diff
@@ -5,6 +5,9 @@
 testing all its methods and error conditions.
 """
 
+# pylint: disable=missing-module-docstring, missing-class-docstring,
```

Comment:
please consider refactoring the code snippets that these rules are highlighting.